docs/integrations/apps/guide/publishing.mdx

```diff
@@ -75,7 +75,7 @@ Provide a helpful description with screenshots and a video to promote your app.
 | App Screenshot | Screenshots of app UI in a BigCommerce store's control panel |
 | Alternate Logo | Used if app featured in Marketplace carousel; should be 259 x 158px (or larger at ratio); color background and light branding; no taglines |
 
-## Add supported features
+## Add supported features 
```

Comment:
@jcnich When app partners submit their app for Marketplace approval in the Developer Portal, how can they let BC know that their app is MLI-compatible? Do they indicate something under the [Add supported features](https://developer.bigcommerce.com/docs/integrations/apps/guide/publishing#add-supported-features) section? When I logged into the developer portal, I didn't see an option for doing this

Reply:
Hey @bc-andreadao yes that's correct - we asked the IAM team to hold off rolling it out to the production Dev Portal until we had the documentation ready, just to try and get those out close to the same time so that the setting wasn't there without an explanation. See ANA-3865 for that work and the feature flag it's behind. I can ask them to enable it anytime though.

Reply:
Quick side note, the feature IS enabled in INT Dev Portal, I can grab a screenshot of it for you if you need to see it.

Reply:
@jcnich Yes, I need it enabled on my store so I can update the Dev Portal screenshot ([Line 82](https://github.com/bigcommerce/docs/pull/366/files#diff-5e01676aa47f71d1b5644a93c7cf9bfacdb3127c8a96065abf0692ffe358420fR82)). I just left a comment in ANA-3865 to get it enabled, so waiting on that

Reply:
@bc-andreadao just as an FYI, it's not a per-store feature, it is for the Dev Portal so it's enabled for all users or none I believe. It is enabled in the INT Dev Portal though, I can help you get there tomorrow on the call.